autoload/mundo.vim

```diff
@@ -385,28 +388,88 @@ function! mundo#MundoHide()"{{{
 endfunction"}}}
 
 function! mundo#MundoRenderGraph()"{{{
+    " Save window view information and buffer number
+    let currentWin  = bufwinnr('%')
+    let winView = winsaveview()
+
     call s:MundoPython('MundoRenderGraph()')
+
+    " Return to buffer and restore window view
+    execute currentWin .'wincmd w'
+    call winrestview(winView)
 endfunction"}}}
 
 " automatically reload Mundo buffer if open
 function! s:MundoRefresh()"{{{
-  " abort when there were no changes
+    " abort if Mundo is closed or cursor is in the preview window
+    let mundoWin    = bufwinnr('__Mundo__')
+    let mundoPreWin = bufwinnr('__Mundo_Preview__')
+    let currentWin  = bufwinnr('%')
+
+    if (mundoWin == -1) || (mundoPreWin == -1) || (mundoPreWin == currentWin)
+        return
+    endif
+
+    " Normal refresh
+    if !get(g:, 'mundo_auto_preview', 0) ||
+                \ get(g:, 'mundo_auto_preview_delay', 0) <= 0
+        return mundo#MundoRenderGraph()
+    endif
 
-  let mundoWin    = bufwinnr('__Mundo__')
-  let mundoPreWin = bufwinnr('__Mundo_Preview__')
-  let currentWin  = bufwinnr('%')
+    " Delayed refresh
+    if !get(g:, 'mundo_enable_inline_delay', 0) ||
```

Comment:
Since `mundo_enable_inline_delay` isn't actually used for any `timer_start` call, it is effectively only a boolean property that determines whether `mundo#MundoRenderGraph()` is called in `MundoRefresh()` or `MundoRefreshDelayed()`. I think you should remove the setting altogether.